### DISCUSSION TITLE
Enh integration of docling library to parse documents

### DISCUSSION THREAD

**Discussion by flefevre** [langflow-ai/langflow#5393] [Ideas]
Sorry for my initial question, do you prefer we create ticket on GitHub or Thread on discord?

I was trying to understand how to integrate the newly IBM library docling able to parse many documents with image, charts etc...
Is it possible to do it with langflow?
What is the best practice?

https://ds4sd.github.io/docling/

https://discord.com/channels/1116803230643527710/1117938353761947678/1320162177537015879

**Comment by flefevre** [comment]
Integrating docling will be a huge update since it is able to parse multiple types of file their images, mathematical formula...
It could replace the library pypdf

**Comment by severfire** [comment]
I am experimenting with it, but I do have issue with some kind of memory leak...
something does not want to release RAM hmm... and I do not know what, docling, langflow or what...
looking into unstructured.io now
